Modular sprite rendering v1

titapo opened the pull request.

titapo committed:
hand with revolver is rendered

titapo committed:
other players' hand is rendered

Until now only the own player's character had hand. The character
building has been unified, but the code is not pleasant.

DomonkosSuranyi commented on client/src/entities/player.rs:
```diff
@@ -1,29 +1,60 @@
+    ) -> Entity
+{
+    let mut hand_transform = Transform::default();
+    hand_transform.set_translation_xyz(-3., -6., 0.);
```
Not has to be changed now just we should think about how could we calculate these offsets from sprite size

titapo replied:
I think that we cannot generate it automatically from the sprite size, but we might store this information separately from the code. (Eg in a ron file). To be padantic: this offset (where is Clint's shoulder) is partially an attribute of the parent, and partially comes from the sprite orientation.

This or that way, storing this information in a ron file would help us to faster align this hierarchies. (It is tedious to rebuild everything for one pixel change)

titapo committed:
visibility order fixed

z coordinates are meant to be interpreted as meters. The hand (children)
coordinates are relative.

titapo commented:
this closes #48 too
